robosuite/environments/sawyer_lift.py

```diff
@@ -826,10 +844,41 @@ def _reset_internal(self):
             radius = np.random.uniform(self._goal_radius_low, self._goal_radius_high)
         assert(radius > self._goal_tolerance)
 
-        cube_pos[0] += radius * np.cos(angle)
-        cube_pos[1] += radius * np.sin(angle)
-        print(cube_pos)
-        self._set_target(pos=cube_pos)
+        target_pos = cube_pos.copy()
+        target_pos[0] += radius * np.cos(angle)
+        target_pos[1] += radius * np.sin(angle)
+        self._set_target(pos=target_pos)
+        for _, o in self.interactive_objects.items():
+            o.reset()
+        # with open('default.xml') as f:
```

Comment:
still need these comments, or should they be deleted?

Reply:
i'll clean these up